renovate.json5

```
@@ -60,9 +60,9 @@
       allowedVersions: '<21.0.0',
     },
     {
-      // TODO(STENCIL-710): Increment this value as a part of updating TypeScript
+      // Increment this value as a part of updating TypeScript
       matchPackageNames: ['typescript'],
-      allowedVersions: '<5.1.6',
+      allowedVersions: '<5.2.2',
```

Since we don't know if there'll be a 5.2.3, what're your thoughts on updating this to something like:
```suggestion
      allowedVersions: '<5.3.0',
```

That way, anything that's 5.2.x can get managed by Renovate?

---

oh yeah that's probably a good idea, I put the exact version in since I thought we usually do all typescript version changes manually but I guess we can probably do patch releases safely

---

Yeah, I think that compounds with the last time we did a bump (5.0->5.1) we were 2 minors behind, so we assumed there wasn't gonna be an additional patch release for 5.1